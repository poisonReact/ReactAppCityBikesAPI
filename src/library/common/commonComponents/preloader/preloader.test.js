import React from 'react'
import renderer from 'react-test-renderer'
import Preloader from 'library/common/commonComponents/preloader/preloader'

it("render correctly preloader component", () => {
    const PreloaderComponent = renderer.create(<Preloader />).toJSON()
    expect(PreloaderComponent).toMatchSnapshot()
})