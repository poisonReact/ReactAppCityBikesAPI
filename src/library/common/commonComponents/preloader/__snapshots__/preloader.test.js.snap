// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly preloader component 1`] = `
<div
  className="loader"
>
  <svg
    height="48"
    version="1.1"
    viewBox="0 0 300 300"
    width="48"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M 150,0 a 150,150 0 0,1 106.066,256.066 l -35.355,-35.355 a -100,-100 0 0,0 -70.711,-170.711 z"
      fill="#76f19a"
    >
      <animateTransform
        attributeName="transform"
        attributeType="XML"
        begin="0s"
        dur="1s"
        fill="freeze"
        from="0 150 150"
        repeatCount="indefinite"
        to="360 150 150"
        type="rotate"
      />
    </path>
  </svg>
</div>
`;
